docs/eso/eso.rst

```diff
@@ -329,7 +350,7 @@ Downloading datasets from the archive
 Continuing from the query with constraints example, the first two datasets are selected,
 using their data product IDs ``DP.ID`` (or ``ARCFILE`` for surveys), and retrieved from the ESO archive.
 
-.. code-block:: python
+.. doctest-skip::
```

Comment:
getting the downloads tested would be OK, just make sure we select a small data file, and cleanup afterwards with `testcleanup`

Reply:
That's not why it's skipped, you need to be logged in to be allowed to download the file.

Reply:
fair enough, I'm sorry for the noise

Reply:
No it's a fair question, I'll add a comment

Reply:
Can't leave a comment in markdown, so this will be the documentation about why this is skipped.